src/Core/Settings.h

```diff
@@ -324,6 +324,7 @@ class IColumn;
     M(Bool, log_profile_events, true, "Log query performance statistics into the query_log and query_thread_log.", 0) \
     M(Bool, log_query_settings, true, "Log query settings into the query_log.", 0) \
     M(Bool, log_query_threads, true, "Log query threads into system.query_thread_log table. This setting have effect only when 'log_queries' is true.", 0) \
+    M(String, log_comment, "comment: ", "Log comment format", 0) \
```

Comment:
It's not a "format", it's the comment itself.

Reply:
So it should be `Log comment`.

Reply:
So it should be `M(String, log_comment, "", "Log comment", 0)`

Reply:
The description should highlight what the setting does and also the purpose of this setting.